main/ui/src/main/java/org/cryptomator/ui/vaultoptions/MountOptionsController.java

```diff
@@ -110,6 +104,18 @@ public void initialize() {
 		);
 	}
 
+	private void enforceToggleState() {
+		if (vault.getVaultSettings().useCustomMountPath().get()
+				&& !Strings.isNullOrEmpty(vault.getVaultSettings().customMountPath().get())
```

Comment:
Won't this prevent the legotimate intermediate state when the user selects "custom mount point" fir the first time, but has't selected a path yet?
Thus immediately switching back to one of the other values, preventing to select a path?

Reply:
The containing method `enforceToggleState()` is only called by `initialize()` and by `chooseCustomMountPoint()`, i.e. it's only called during the initialization of the window (where it's crucial that the correct option is selected automatically) and **after** choosing a location. 
Also the same technique is used for the selection of the drive letter, see: https://github.com/cryptomator/cryptomator/blob/30c3c2cbe688ef7c470f4e7ceec85c00d0431e04/main/ui/src/main/java/org/cryptomator/ui/vaultoptions/MountOptionsController.java#L112

Reply:
I see.

Reply:
Fixed by 4c844872da94384be570d48d67075da42d6cab51.